lib/src/commands/music.dart

```diff
@@ -0,0 +1,161 @@
+import 'package:nyxx/nyxx.dart';
+import 'package:nyxx_commands/nyxx_commands.dart';
+import 'package:running_on_dart/src/checks.dart';
+import 'package:running_on_dart/src/services/music.dart';
+
+ChatGroup music = ChatGroup(
+  'music',
+  'Music related commands',
+  checks: [
+    GuildCheck.all(),
+    sameVoiceChannelOrDisconnectedCheck,
+  ],
+  children: [
+    ChatCommand(
+      'play',
+      'Plays music based on the given query',
+      id('music-play', (
+          IChatContext context,
+          @Description('The name/url of the song/playlist to play') String query
+        ) async {
+        final node = MusicService.instance.cluster.getOrCreatePlayerNode(context.guild!.id);
+        connectIfNeeded(context);
+        final result = await node.autoSearch(query);
+
+        if (result.tracks.isEmpty) {
+          await respond(context, MessageBuilder.content('No results were found'));
```

Comment:
Simply using `context.respond` is enough, no need for a custom function.

Reply:
If `context.respond` is always used, it results in something like this when using the commands as slash ones
https://cdn.discordapp.com/attachments/985589157088874506/986725857303359618/unknown.png

So i made that function because i want it to look like this:
https://cdn.discordapp.com/attachments/985589157088874506/986726121552879717/unknown.png

Reply:
This only occurs because of high latency when auto acknowledge kicks in - this will not be an issue when released.

In fact, since most commands in this bot have no hidden responses, you could pass `hideOriginalResponse: false` to `CommandsOptions` to make the original response public.

The behaviour you observed is intentional.

Reply:
From the docs:
> While the hidden and private arguments are guaranteed to hide/show the resulting response, slow commands might sometimes show strange behaviour in their responses. Acknowledging the interaction early with the correct value for hidden can prevent this behaviour.